lib/cloudant.js

```diff
@@ -49,11 +51,17 @@ function Cloudant(settings, ds) {
     throw new Error('Invalid settings: "url" OR "username"' +
     ' AND "password" required');
   }
-  this.creds = settings.url || {
-    account: settings.username,
-    password: settings.password,
-  };
-  this.cloudant = Driver(this.creds);
+
+  if (settings.url) {
+    this.options = settings.url;
```

Comment:
If always constructing an object and just appending the properties is valid, I can just do that.